src/main/java/at/hannibal2/skyhanni/config/features/inventory/helper/HelperConfig.java

```diff
@@ -13,6 +13,16 @@ public class HelperConfig {
     public HarpConfig harp = new HarpConfig();
 
     public static class HarpConfig {
+        @Expose
+        @ConfigOption(name = "GUI Scale", desc = "Automatically sets the GUI scale to \"AUTO\" when entering the Harp")
+        @ConfigEditorBoolean
+        @FeatureToggle
+        public boolean guiScale = false;
+        @Expose
+        @ConfigOption(name = "Quick Restart", desc = "When pressing the close Button in the Harp Menu shortly after opening the Harp the selected song will be started")
+        @ConfigEditorBoolean
+        @FeatureToggle
+        public boolean quickRestart = false;
```

Comment:
These don't need a feature toggle
Also missing the empty lines between the options in this file